.github/workflows/azure-static-web-apps.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         id: builddeploy
         uses: Azure/static-web-apps-deploy@v1
         with:
+          skip_deploy_on_missing_secrets: true
```

Comment:
does this prevent failure for dependabot pull requests?

Reply:
No idea yet, will let you know shortly. Just want to reduce the number of dependabot PRs quick

Reply:
Don't know if this works. But I did find how I could give dependabot the deployment token in the repo settings, which looks like it worked